docs/ftldns/configfile.md

```diff
@@ -56,6 +56,32 @@ How should `FTL` reply to blocked queries?<br>
 #### `DBFILE=/etc/pihole/pihole-FTL.db` {#dbfile data-toc-label='DB File'}
   Specify path and filename of FTL's SQLite3 long-term database. Setting this to `DBFILE=` disables the database altogether
 
+### File options
+
+#### `LOGFILE=/var/log/pihole-FTL.log` {#file_LOGFILE data-toc-label='Log file'}
+  Location of FTL's log file.
+
+#### `PIDFILE=/var/run/pihole-FTL.pid` {#file_PIDFILE data-toc-label='Process identifier file'}
```

Comment:
Note that we cannot use `PID file` as TOC label here.
`mkdocs` fails to parse this string as it tries to put the tooltip under `PID` causing everything to fail...

![Screenshot from 2019-06-10 11-40-20](https://user-images.githubusercontent.com/16748619/59186990-9a239080-8b74-11e9-8afd-94b2ad1e2304.png)